mozilla_django_oidc_db/models.py

```diff
@@ -250,7 +250,11 @@ class OpenIDConnectConfig(OpenIDConnectConfigBase):
     username_claim = ClaimField(
         verbose_name=_("username claim"),
         default=ClaimFieldDefault("sub"),
-        help_text=_("The name of the OIDC claim that is used as the username"),
+        help_text=_(
+            "The name of the OIDC claim that is used as the username"
+            "To access nested claims, use a list of keys that will be joined with a dot. "
+            "For example, ['userinfo', 'email'] accesses the claim 'userinfo.email'."
+        ),
```

Comment:
Using the dot notation might be confusing, so let's make this as explicit as possible
```suggestion
        help_text=_(
            "The name of the OIDC claim that is used as the username. "
            "To access nested claims, use a list of keys that will be joined with a dot. "
            "For example, ['userinfo', 'email'] accesses the claim {"userinfo": {"email": "foo@bar.com"}}. "
            "To access claims with dots in the name, you can specify the name as such. "
            "For example, ['userinfo.email'] accesses the claim {"userinfo.email": "foo@bar.com"}"
        ),
```